test/unit/services/storage/models.spec.ts

```diff
@@ -332,6 +333,41 @@ describe('Models', () => {
       ]
       expect(offers).to.be.deep.equal(expectedRes)
     })
+    it('should aggregate available size for offers', async () => {
```

Comment:
Also, can you please add test for retrieving min/max value for `availableCapacity`